lang.go

```diff
@@ -465,6 +470,8 @@ func LangToString(lang Lang) string {
 		return "ces"
 	case Cmn:
 		return "cmn"
+	case Yue:
```

Comment:
The languages are in order alphabetically (in English). These lines should go above `case Zul:`.